types/ckeditor__ckeditor5-pagination/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for @ckeditor/ckeditor5-export-word 27.0
```

Comment:
```suggestion
// Type definitions for @ckeditor/ckeditor5-export-word 27.1
```
https://www.npmjs.com/package/@ckeditor/ckeditor5-pagination
?